### ISSUE TITLE
Add additional installation steps in tutoria l(AutoMM Detection - Quick Start on a Tiny COCO Format Dataset)

### ISSUE THREAD

**Issue by afmkt** [autogluon/autogluon#3963]
### Describe the issue linked to the documentation

When I followed "AutoMM Detection - Quick Start on a Tiny COCO Format Dataset" in my windows box, I encountered two errors about missing library mmcv and missing DLL nvml in step "Finetuning the Model".

### Suggest a potential alternative/fix

add additional instructions in section ""Finetuning the Model"" to avoid the annoying errors.
```
!pip install -U openmim
!mim install mmcv
!pip install pynvml
```

**Comment by cheungdaven** [comment]
Hi @FANGAreNotGnu , could you take a look? Thank you!

**Comment by FANGAreNotGnu** [comment]
Fixed in [#4430](https://github.com/autogluon/autogluon/pull/4430). pynvml is not listed as it's no more needed as dependency.
